gpt_engineer/core/chat_to_files.py

```diff
@@ -134,21 +140,25 @@ def parse_diffs(diff_string: str) -> dict:
     - dict: A dictionary of Diff objects keyed by filename.
     """
     # Regex to match individual diff blocks
-    diff_block_pattern = re.compile(
+    diff_block_pattern = regex.compile(
```

Comment:
It is great that you are improving the diff-pipeline! Formally, this should be a separate PR. Would you be able to make a separate PR with the changes to chat_to_files?

Reply:
I can merge that quickly

Reply:
Will do

Reply:
@ATheorell https://github.com/gpt-engineer-org/gpt-engineer/pull/1067